Documentation/git-cherry-pick.txt

```diff
@@ -11,6 +11,7 @@ SYNOPSIS
 'git cherry-pick' [--edit] [-n] [-m parent-number] [-s] [-x] [--ff]
 		  [-S[<keyid>]] <commit>...
 'git cherry-pick' --continue
+'git cherry-pick' --skip
 'git cherry-pick' --quit
 'git cherry-pick' --abort
```

Comment:
Instead of having four single-command lines here, it might be nicer to have
  'git cherry-pick' [--continue | --skip | --abort | --quit]
which I think is the way rebase handles it.